### DISCUSSION TITLE
How to toggle floating diagnostics?

### DISCUSSION THREAD

**Discussion by GreenSleeper** [LazyVim/LazyVim#1126] [Q&A]
I'm working with an large codebase, there are all kinds of files in it. And for C/C++, I can't get a complete compile database for every file, so there will always be a lot of errors and warnings with clangd, it's distracting.

How can I define a keymap that toggles floating diagnostics virtual text.

Or maybe suppress it when open nvim, just leave the symbols leftside.

**Comment by folke** [comment]
`<leader>ud`
